source/lib.ts

```diff
@@ -59,13 +71,8 @@ async function handledDroppedPermissions({origins}: chrome.permissions.Permissio
 		return;
 	}
 
-	for (const [origin, scriptPromise] of registeredScripts) {
-		if (origins.includes(origin)) {
-			// eslint-disable-next-line no-await-in-loop
-			const script = await scriptPromise;
-			void script.unregister();
```

Comment:
Pretty sure this did not work in event pages and service workers. Nothing happened then, they'd just be left behind but never injected.